fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -457,6 +459,17 @@ protected FedoraResource resource() {
         return resource;
     }
 
+    protected void addTimeMapHeader(final FedoraResource resource) {
+        if (resource instanceof FedoraTimeMap) {
+            final URI parentUri = getUri(resource());
+            servletResponse.addHeader(LINK, Link.fromUri(VERSIONING_TIMEMAP_TYPE).rel("type").build().toString());
+            servletResponse.addHeader(LINK, Link.fromUri(parentUri).rel("original").build().toString());
```

Comment:
@lsitu Can I get you to add a Link for `rel="timegate"` also to the `parentUri` and one to `rel="timemap"` to itself.

Reply:
@whikloj The header link rel="timegate" is added and along with it is delete option that seems to be missing ...

Reply:
@whikloj Link header rel="timemap" is also added.